server/src/main/java/org/elasticsearch/index/SlowLogLevel.java

```diff
@@ -21,8 +21,16 @@
 import java.util.Locale;
 
 public enum SlowLogLevel {
-    WARN, TRACE, INFO, DEBUG;
+    WARN, //most specific - little logging
+    INFO,
+    DEBUG,
+    TRACE; //least specific - lots of logging
     public static SlowLogLevel parse(String level) {
         return valueOf(level.toUpperCase(Locale.ROOT));
     }
+
+    boolean isLevelEnabledFor(SlowLogLevel levelToBeUsed) {
+        // info is less specific then warn
+        return this.ordinal() >= levelToBeUsed.ordinal();
```

Comment:
Agree - this was a shortcut that should be avoided. I will add a field.

Reply:
I am  also  not sure about the method name. 
I wanted to avoid `isMoreSpecificThan` or `isLessSpecificThan` (used in log4j) as they don't seem to be too  obvious to me.
Do you think we should stick with `isLevelEnabledFor` for now? any other ideas?

Reply:
I agree, the `isMoreSpecificThan` and `isLessSpecificThan` naming is not great. I like your choice of `isLevelEnabeldFor`.